frontend/src/site/ProductsSite.tsx

```diff
@@ -0,0 +1,30 @@
+import ProductCard from "../component/ProductCard";
```

Comment:
konnte am Saw icon nicht reviewn . Warum nicht in die Cloud hochgeladen und von dort genutzt